src/signIn.ts

```diff
@@ -63,6 +64,71 @@ export const getAuthorizeUri = async (
   return `${uri}?${queryParams.join('&')}`
 }
 
+export const signInSilentlyIframeId = (config: OauthClientConfig): string => {
+  const configHash = btoa(
+    `${config.issuer}-${config.clientId}-${(config.scopes || []).join('_')}`
+  )
+  return `lionel-signin-silently-${configHash}`
+}
+
+const _createHandleSigninSilenPostMessageFn =
```

Comment:
Missed a letter I think :) `_createHandleSigninSilenPostMessageFn` → `_createHandleSignInSilentPostMessageFn`

Reply:
Nice catch! I'll fix this!